chef/data_bags/crowbar/migrate/glance/012_add_store_admin_keyring.rb

```diff
@@ -0,0 +1,9 @@
+def upgrade ta, td, a, d
+  a['rbd']['store_admin_keyring'] = ta['rbd']['store_admin_keyring']
```

Comment:
This default value for internal ceph deployment, I would prefer to leave it as it is.